### ISSUE TITLE
XBox S/X Controller Won't Pair With a Different Aquarius+

### ISSUE THREAD

**Issue by 1stage** [fvdhoef/aquarius-plus#32]
XBox s/x controller paired with a rev2 prototype Aquarius+, but would not pair with a second, different Aquarius+ signature edition. It would pair with a Windows PC, and when the original rev2 prototype was turned on, it paired back with it as well, but would never pair with the second Aquarius+.

**Comment by 1stage** [comment]
All systems tested were running V0.17a

**Comment by fvdhoef** [comment]
Please retest if running all systems on V0.17 (without 'a') makes a difference.

**Comment by 1stage** [comment]
Downgraded both systems to V0.17. At first, it would not pair with the "new" system. I unplugged and installed the "old" system, forced a re-pairing while old Aq+ was on, then immediately turned it off. Re-plugged-in the "new" system and was able to pair it.

Not sure if it was a V0.17a issue, or oddities with paired device memory.

**Comment by 1stage** [comment]
Issue came back on the "new" unit. Controller, blinks, appears to pair, and then loses conneciton and goes back to blinking again. Went back to V0.17 to confirm, and it still happens.

Seems to only want to pair with the original Aq+.

**Comment by 1stage** [comment]
Now on V0.22a. Even after an esp:settings > sysreset, I can't get an XBox s/x controller previously paired with another aqplus to pair with another, different aqplus. Controller pairs with a Windows 11 PC and an actual Xbox Series S game console.

I've tried on two different Aq+ Minis and on two different Aq+ standard units to no avail. I've tried fresh batteries, I've tried "resetting" the controller (power on, go into pairing mode, force power off, try to re-pair).
